assets/src/infrastructure/ui/display/steps/Step.tsx

```diff
@@ -14,24 +14,25 @@ const Step: React.FC<StepProps> = ({
 	vertical = false,
 	...props
 }) => {
-	const className = classNames(
-		props.className,
-		'ee-form-step',
-		{
-			'ee-form-step--active': active,
-			'ee-form-step--disabled': disabled,
-			'ee-form-step--vertical': vertical,
-			'ee-form-step--inline': !vertical,
-			'ee-form-step--no-icon': !Icon,
-		}
-	);
+	const className = classNames(props.className, 'ee-form-step', {
+		'ee-form-step--active': active,
+		'ee-form-step--disabled': disabled,
+		'ee-form-step--vertical': vertical,
+		'ee-form-step--inline': !vertical,
+		'ee-form-step--no-icon': !Icon,
+	});
```

Comment:
or using short circuit evaluation (if we want to always follow that pattern):

```suggestion
	const className = classNames(
		props.className,
		'ee-form-step',
		active && 'ee-form-step--active',
		disabled && 'ee-form-step--disabled',
		vertical && 'ee-form-step--vertical',
		!vertical && 'ee-form-step--inline',
		!Icon && 'ee-form-step--no-icon',
	});
```

although that suddenly doesn't look as clear for some reason 🤔 
maybe because `ee-form-step--` lined up nicely in the current version

anyways, not going to push this approach as what is there now is 100% correct

Reply:
thx for the suggestion :bow:

Reply:
done